src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+$font-family: Roboto, sans-serif;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$main-width: 166px;
+
 body {
   margin: 0;
 }
+
+@import './style/card';
+@import './style/stars';
```

Comment:
Ensure that the paths in the `@import` statements are correct and that the referenced SCSS files (`card` and `stars`) exist in the specified directory. If the paths are incorrect, it will lead to a compilation error.